yaksh/templates/yaksh/view_answerpaper.html

```diff
@@ -102,7 +103,56 @@
         {% else %}
             <div class="panel panel-danger">
         {% endif %}
-    <div class="panel-heading">Autocheck: {{ answer.error }}</div>
+    <div class="panel-heading">Error:</div>
```

Comment:
So this same html is used in number of templates, so can we have a custom tag for this or any other solution to prevent redundancy.